Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -426,6 +426,20 @@
       components:
       - SurplusBundle
 
+- type: listing
+  id: uplinkMimeBook
+  description: The classical two part series on how to further hone your mime skills.
+  productEntity: MimeSpellbook
+  cost:
+    Telecrystal: 20
```

Comment:
Is this even remotely worth 20 TC?  The gun is alright, but only 3 shots. The wall is good, but i don't think that it's good enough on its own to be worth this much.